src/plugins/workspace/public/components/workspace_form/data_source_connection_table.tsx

```diff
@@ -152,19 +107,25 @@ export const DataSourceConnectionTable = ({
         ]
       : []),
     {
-      width: '25%',
+      width: '20%',
       field: 'name',
       name: i18n.translate('workspace.detail.dataSources.table.title', {
         defaultMessage: 'Title',
       }),
       truncateText: true,
       render: (name: string, record) => {
         const origin = window.location.origin;
+        const basePath = http.basePath.serverBasePath;
         let url: string;
         if (record.connectionType === DataSourceConnectionType.OpenSearchConnection) {
-          url = `${origin}/app/dataSources/${record.id}`;
+          url = http.basePath.prepend(`/app/dataSources/${record.id}`, {
+            withoutClientBasePath: true,
+          });
```

Comment:
```suggestion
          url = http.basePath.prepend(`/app/dataSources/${record.id}`);
```
If it's opensearch connection, shall we just go to the data source page within the workspace?

Reply:
I think we should go to the data source management page within the workspace. @yubonluo , could you help clarify this?